README.md

```diff
@@ -62,6 +70,17 @@ This is a homemade solution that gives us the capacity to controll the AKS nodes
 
 Kured is a simple solution to the problem on rebooting nodes to enable security patches. If an update requires the node to restart, it creates a file that Kured looks for. If the file is there, Kured drains, restart, and then uncordons the given node.
 
+### Loki
+
+Loki allows us to collect logs from both in-cluster pods and VM's outside the cluster. We forward the logs to Grafana where we can centrally search and correlate metrics and logs.
```

Comment:
```suggestion
Loki allows us to collect logs from both in-cluster pods and VM's outside the cluster. We use Grafana to centrally search and correlate metrics and logs stored in Loki.
```

Reply:
prometheus metrics are not stored in Loki

Original is more accurate.